vmware/ngc/NGC-Plugin/pom.xml

```diff
@@ -1,130 +1,209 @@
+			<groupId>com.google.code.gson</groupId>
+			<artifactId>gson</artifactId>
+			<version>2.3.1</version>
+			<systemPath>${env.VSPHERE_SDK_HOME}/libs/gson-2.3.1.jar</systemPath>
 			<scope>system</scope>
 		</dependency>
 		<dependency>
-		    <groupId>javax.servlet</groupId>
-		    <artifactId>servlet-api</artifactId>
-		    <version>3.0.1</version>
-		    <systemPath>${env.VSPHERE_SDK_HOME}/libs/javax.servlet-api-3.0.1.jar</systemPath>
+			<groupId>javax.servlet</groupId>
+			<artifactId>servlet-api</artifactId>
+			<version>3.0.1</version>
+			<systemPath>${env.VSPHERE_SDK_HOME}/libs/javax.servlet-api-3.0.1.jar</systemPath>
 			<scope>system</scope>
 		</dependency>
 		<dependency>
-		    <groupId>org.apache.commons</groupId>
-		    <artifactId>com.springsource.org.apache.commons.lang</artifactId>
-		    <version>2.5.0</version>
-		    <systemPath>${env.VSPHERE_SDK_HOME}/libs/com.springsource.org.apache.commons.lang-2.5.0.jar</systemPath>
+			<groupId>org.apache.commons</groupId>
+			<artifactId>com.springsource.org.apache.commons.lang</artifactId>
+			<version>2.5.0</version>
+			<systemPath>${env.VSPHERE_SDK_HOME}/libs/com.springsource.org.apache.commons.lang-2.5.0.jar</systemPath>
 			<scope>system</scope>
 		</dependency>
 		<!-- https://mvnrepository.com/artifact/org.apache.maven.plugins/maven-dependency-plugin -->
 		<dependency>
-		    <groupId>org.apache.maven.plugins</groupId>
-		    <artifactId>maven-dependency-plugin</artifactId>
-		    <version>3.1.1</version>
+			<groupId>org.apache.maven.plugins</groupId>
+			<artifactId>maven-dependency-plugin</artifactId>
+			<version>3.1.1</version>
 		</dependency>
 		<dependency>
-		    <groupId>org.opensds.vmware</groupId>
-			 <artifactId>NGC</artifactId>
+			<groupId>org.opensds.vmware</groupId>
+			<artifactId>NGC</artifactId>
 			<version>0.0.1-SNAPSHOT</version>
-		    <systemPath>${basedir}/lib/ngc-adapter-manager.jar</systemPath>
+			<systemPath>${basedir}/lib/ngc-adapter-manager.jar</systemPath>
 			<scope>system</scope>
 		</dependency>
-    </dependencies>
-
+		<dependency>
+			<groupId>junit</groupId>
+			<artifactId>junit</artifactId>
+			<version>4.12</version>
+			<scope>test</scope>
+		</dependency>
+		<dependency>
+			<groupId>org.springframework</groupId>
+			<artifactId>spring-test</artifactId>
+			<version>2.5</version>
+			<scope>test</scope>
+		</dependency>
+		<dependency>
+			<groupId>org.springframework</groupId>
+			<artifactId>spring-orm</artifactId>
+			<version>${springVersion}</version>
+			<scope>compile</scope>
+		</dependency>
+		<dependency>
+			<groupId>org.springframework</groupId>
+			<artifactId>spring-webmvc</artifactId>
+			<version>${springVersion}</version>
+			<scope>compile</scope>
+		</dependency>
+		<dependency>
+			<groupId>org.springframework</groupId>
+			<artifactId>spring-oxm</artifactId>
+			<version>${springVersion}</version>
+			<scope>compile</scope>
+		</dependency>
+		<dependency>
+			<groupId>junit</groupId>
```

Comment:
junit repeated?

Reply:
Yes, it was merge from previos PR. Removed now.